code/modules/spacepods/spacepod.dm

```diff
@@ -378,12 +378,17 @@
 					"<span class='notice'>You start drilling through the [src]'s lock!</span>")
 				if(do_after(user, 100 * W.toolspeed, target = src))
 					QDEL_NULL(equipment_system.lock_system)
+					unlocked = TRUE
 					user.visible_message(user, "<span class='warning'>[user] has destroyed the [src]'s lock!</span>",
 						"<span class='notice'>You destroy the [src]'s lock!</span>")
 				else
 					user.visible_message(user, "<span class='warning'>[user] fails to break through the [src]'s lock!</span>",
 					"<span class='notice'>You were unable to break through the [src]'s lock!</span>")
 				return
+			if(L.on && unlocked == FALSE) //The buster is on, we don't have a lock system, and the pod is still somehow locked, unlocking.
+				unlocked = TRUE
+				user.visible_message(user, "<span class='warning'>[user] has destroyed the [src]'s malfunction that locks the doors!</span>",
```

Comment:
This is some _really_ awkward phrasing. "John Wick has destroyed the the space pod's malfunction that locks the doors!"
It'd be better as, like, `<span class='notice'>[user] repairs [src]'s doors with [L].</span>`